packages/models-library/src/models_library/api_schemas_webserver/socketio.py

```diff
@@ -0,0 +1,17 @@
+from ..users import GroupID, UserID
+
+
+class SocketIORoom(str):
```

Comment:
I would suggest to name as `SocketIORoomStr` to follow convention of all str-based types (see models_library)

Is here any reason why not to inherit from a constraint str as `IDStr`?

SEE https://github.com/itisfoundation/osparc-simcore/blob/5f4a44ce95db4ce6f713e04be1bbde426622af8a/packages/models-library/src/models_library/basic_types.py#L75-L80